Automattic/wp-calypso
# Components: Cleanup obsolete/duplicate IDs from JetpackHeader

Currently, we have a lot of unnecessary `id`s in the SVGs of `<JetpackHeader />`, some of them are also duplicate, which causes the svgs to be invalid.

#### Changes proposed in this Pull Request

* Cleanup obsolete or duplicate `id`s from `<JetpackHeader />`

#### Testing instructions

* Checkout this branch, or spin it up at calypso.live
* Go to `/devdocs/design/jetpack-header`
* Compare with https://wpcalypso.wordpress.com/devdocs/design/jetpack-header
* Verify the component demos look and are positioned the same way on both locations.
